subsys/bluetooth/services/fast_pair/fp_storage/include/fp_storage_pn.h

```diff
@@ -21,14 +21,20 @@ extern "C" {
 #endif
 
 /** Length of buffer used to store Personalized Name. */
+#ifdef CONFIG_BT_FAST_PAIR_STORAGE_EXT_PN_LEN_MAX
 #define FP_STORAGE_PN_BUF_LEN	(CONFIG_BT_FAST_PAIR_STORAGE_EXT_PN_LEN_MAX + 1)
+#else
+#define FP_STORAGE_PN_BUF_LEN	0
+#endif
 
 /** Save Personalized Name.
  *
- * @param[in] pn_to_save Personalized Name to be saved. Length of Personalized Name string can not
- *			 be greater than @ref CONFIG_BT_FAST_PAIR_STORAGE_EXT_PN_LEN_MAX.
+ * @param[in] pn_to_save Personalized Name to be saved.
  *
- * @return 0 If the operation was successful. Otherwise, a (negative) error code is returned.
+ * @retval 0 If the operation was successful.
+ * @retval -ENOMEM If length of Personalized Name string is greater than
+ *	   @ref CONFIG_BT_FAST_PAIR_STORAGE_EXT_PN_LEN_MAX.
+ * @retval Negative error code otherwise.
```

Comment:
`@returnNegative error code otherwise.`
retval should be used only with explicit error codes.